.github/workflows/behat-stability-1-upgrade-path.yml

```diff
@@ -79,6 +79,11 @@ jobs:
       - name: Run the updates
         run: docker exec -i social_ci_web bash /var/www/scripts/social/install/update.sh
 
+      - name: Restart Docker containers
```

Comment:
@Ressinel Why do we need to restart the containers?

Reply:
@Ressinel See the output of restarting container here: https://github.com/goalgorilla/open_social/runs/7558047207?check_suite_focus=true#step:14:1 you will see that the containers are getting re-created.

Reply:
@navneet0693 Actually, I am not completely sure why it doesn't work without restarting containers. But it is only one decision when the upgrade path works with reproducible error.